src/System.Linq/tests/Performance/Perf.Linq.cs

```diff
@@ -42,6 +42,29 @@ public static IEnumerable<object[]> IterationSizeWrapperDataNoWrapper()
                     yield return new object[] { size, iteration };
         }
 
+        private static int[] QuiskSortWorstCase(int n)
+        {
+            int OddPos(int k)
+            {
+                var s = k;
```

Comment:
Nit: in general, please search and replace var with the proper type anywhere it's not explicitly stated on the RHS